src/cart/crt_init.c

```diff
@@ -275,6 +275,12 @@ crt_init_opt(crt_group_id_t grpid, uint32_t flags, crt_init_options_t *opt)
 		D_GOTO(out, rc);
 	}
 
+	rc = d_mm_init(9);
+	if (rc && rc != -DER_ALREADY) {
+		D_ERROR("dm_init() failed, rc: %d.\n", rc);
```

Comment:
DP_RC, DF_RC.  The function is d_mm_init(), but the log is dm_init()

Reply:
done